helios-tools/src/main/java/com/spotify/helios/cli/command/JobCreateCommand.java

```diff
@@ -532,6 +550,17 @@ int run(final Namespace options, final HeliosClient client, final PrintStream ou
       builder.setToken(token);
     }
 
+    final Set<String> addCapabilities = ImmutableSet.copyOf(
+        options.<String>getList(addCapabilityArg.getDest()));
+    if (!addCapabilities.isEmpty()) {
+      builder.setAddCapabilities(addCapabilities);
```

Comment:
if the setAddCapabilities method took a Collection instead of a Set, you could just pass the list, and rely on `ImmutableSet.copyOf(collection)` in the method to do the type conversion